lasagne/layers/merge.py

```diff
@@ -6,7 +6,8 @@
 __all__ = [
     "ConcatLayer",
     "concat",
-    "ElemwiseSumLayer",
+    "ElemwiseMergeLayer",
+    "ElemwiseSumLayer"
```

Comment:
Ok. I have been wondering about the wiered commas :) thanks for the explanation.